build-system/tasks/serve.js

```diff
@@ -213,5 +219,7 @@ serve.flags = {
   esm: '  Serve ESM JS (requires the use of --new_server)',
   cdn: '  Serve current prod JS',
   rtv: '  Serve JS from the RTV provided',
-  coverage: '  Serve instrumented code to collect coverage info',
+  coverage:
+    '  Serve instrumented code to collect coverage info; use ' +
+    '--compiled=live to auto-report coverage on page unload',
```

Comment:
Typo: This should be `--coverage=live`, not `--compiled=live`.

Reply:
Good catch! Thanks. Fixed.